main.ts

```diff
@@ -13,8 +13,11 @@ enum RcbPowerLevel {
     No
 }
 
-//% color="#9C5F9B" block="Roller Coaster" icon="\uf3ff"
+//% color="#9C5F9B" block="Roller Coaster Test" icon="\uf3ff"
```

Comment:
Sorry I missed this yesterday. Wondering if this "test" is supposed to still be here?

Reply:
Whoops, good catch!

Reply:
(Fixed it with https://github.com/microsoft/makecode-minecraft-roller-coaster/commit/d231cbd097bfb1c9f83a1d64ba1335965d7861b1)